include/clang/Basic/CodeGenOptions.def

```diff
@@ -364,6 +364,9 @@ CODEGENOPT(BranchTargetEnforcement, 1, 0)
 /// Whether to emit unused static constants.
 CODEGENOPT(KeepStaticConsts, 1, 0)
 
+/// Whether to add null ptr checks for checkedc.
```

Comment:
Do we want the nullptr checks to be emitted by default?

Reply:
Yes, we want to emit them by default. Please see @dtarditi 's comment above.